exercises/01-composite-ui/before/Divergent.Customers.API/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Microsoft.Owin.Hosting;
+
+namespace Divergent.Customers.API
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            using (WebApp.Start<Startup>("http://localhost:20186"))
+            {
+                Console.WriteLine("Dicergent.Customers.API web server is running.");
+                Console.WriteLine("Press <enter> to quit.");
+                Console.ReadLine();
```

Comment:
I'm not sure this is best. The NSB endpoints in the later exercises require <kbd>Ctrl</kbd>+<kbd>C</kbd> which I think is better, to avoid accidental shutdowns:

https://github.com/Particular/Workshop/blob/9db8434954ee388a47cccad7d3520e251e06fb25/exercises/02-publish-subscribe/after/Divergent.Customers/Program.cs#L28-L31